dev-python/urlgrabber/urlgrabber-4.1.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+inherit distutils-r1
+
+DESCRIPTION="Python module for downloading files"
+HOMEPAGE="http://urlgrabber.baseurl.org https://github.com/rpm-software-management/urlgrabber"
+SRC_URI="https://github.com/rpm-software-management/${PN}/archive/${PN}-${PV//./-}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ia64 ~ppc ~ppc64 ~x86"
+
+RDEPEND="
+	dev-python/pycurl[${PYTHON_USEDEP}]
+	dev-python/urllib3[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+
+S="${WORKDIR}/${PN}-${PN}-${PV//./-}"
+
+# Entire testsuite relies on connecting to the i'net
```

Comment:
Please add the support for it and `RESTRICT=test`, so people could run it as a last resort.

Reply:
@mgorny , added restricted test phase and some patches to run the unittests without failure
also added a [bug](https://github.com/rpm-software-management/urlgrabber/issues/29) upstream for the failing unittest